### ISSUE TITLE
6.0 Feedback:  Experience > Getting to deck calibration with an uncalibrated robot is not obvious

### ISSUE THREAD

**Issue by y3rsh** [Opentrons/opentrons#10476]
### Overview

<img width="800" alt="image" src="https://user-images.githubusercontent.com/502770/170565241-4da448bf-4d4c-4fa3-9394-5bdf29e084cf.png">

### Expected Behavior

Could the alerts launch deck calibration?
Could there be an alert at top to launch deck calibration? 

### Operating system

Mac

**Comment by nusrat813** [comment]
@emilywools thoughts on this

**Comment by emilywools** [comment]
you're right that this is not a good experience. Morgan just wrapped up a project to make robot calibration much more clear, but its not in scope for 6.0

**Comment by nusrat813** [comment]
In scope for 6.2.0 - https://www.figma.com/file/8R6jZdYMMzAh6aeRkzHz4l/New-user-calibration?node-id=1598%3A114042
